src/components/auth/ForgotPasswordForm.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
Still see some files with semicolons so please make sure you have formatted all files accordingly. 
You can format all files at once https://stackoverflow.com/a/57629631